src/SFA.DAS.EmploymentCheck.Functions/AzureFunctions/Orchestrators/CreateEmploymentCheckCacheRequestsOrchestrator.cs

```diff
@@ -58,5 +73,124 @@ public class CreateEmploymentCheckCacheRequestsOrchestrator
                 context.ContinueAsNew(null);
             }
         }
+
+        public bool IsValidEmploymentCheckData(EmploymentCheckData employmentCheckData)
```

Comment:
This is a lot of logic for an orchestrator. I wonder if it makes more sense to extract all this validation into another class

Reply:
Moved the validation logic to a separate EmploymentCheckDataValidator class.